src/emulator/config.cpp

```diff
@@ -163,7 +163,7 @@ static ExitCode parse(Config &cfg, fs::path load_path, const std::string &root_p
     get_yaml_value(config_node, "sys-lang", &cfg.sys_lang, static_cast<int>(SCE_SYSTEM_PARAM_LANG_ENGLISH_US));
     get_yaml_value(config_node, "background-image", &cfg.background_image, std::string{});
     get_yaml_value(config_node, "background-alpha", &cfg.background_alpha, 0.300f);
-    get_yaml_value(config_node, "log-level", &cfg.log_level, static_cast<int>(spdlog::level::trace));
+    get_yaml_value(config_node, "log-level", &cfg.log_level, static_cast<int>(spdlog::level::info));
```

Comment:
remove

Reply:
why remove ? i have set it for normal user for get info byu default, like rpcs3 on succes, can change it after, is just default value for normal user

Reply:
No. We want as much info as possible.
Even it was an acceptable change, why even include it in the same commit???

Reply:
ah yes i see probleme artf, i have thinks works like rpcs3, but here set info is not oinly for showing log, but only for print inside log file arf
i have removed